templates/flows/flow_editor.haml

```diff
@@ -386,6 +386,7 @@
 
                     .node-attachment.viewer-button{ng-class:'{"icon-videocam": action._media.type == "video", "icon-photo": action._media.type=="image", "icon-arrow-right-8": action._media.type=="audio"}', ng-show:'action._media', ng-click:'clickShowActionMedia(action); $event.stopPropagation();'}
                     .node-attachment.icon-attachment{ng-show:'action._attachURL'}
+                    .node-attachment.viewer-button{ng-class:'{"icon-button-quick-reply": action.quick_replies.length > 0}', ng-show:'action.quick_replies.length > 0'}
```

Comment:
css class doesn't need to be conditional here, i.e.

`.node-attachment.viewer-button.icon-button-quick-reply`